src/Exscript/protocols/SSH2.py

```diff
@@ -255,7 +255,7 @@ def _paramiko_shell(self):
             self._dbg(1, 'Failed to open shell.')
             raise LoginFailure('Failed to open shell: ' + str(e))
 
-    def _connect_hook(self, hostname, port):
+    def _connect_hook(self, hostname, port, init_timeout):
```

Comment:
You added an unused parameter. Probably to be compatible with Telnet implementation. But I really don't like that.
If it's there, it should do something. I would remove it anyway (see general pull request comment).